src/content/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model.mdx

```diff
@@ -110,18 +110,22 @@ For tips on editing user type, see [Manage user type](#update-user-type).
     id="update-user-type"
     title="Manage user type and user upgrades"
   >
-    Some important tips for managing user type: 
-    * If you're on the [New Relic One pricing model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models), your users' [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type) is a billing factor. 
-    * If you change a user's type to a higher type (either via the UI or via an upgrade request email), their user type is changed to that higher user type across all accounts in the organization. 
+Some important tips for managing user type: 
+  * If you're on the [New Relic One pricing model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models), your users' [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type) is a billing factor. 
+  * If you change a user's type to a higher type (either via the UI or via an upgrade request email), their user type is changed to that higher user type across all accounts in the organization. 
```

Comment:
Is there another way to say "higher type" in this sentence for a smoother reading experience? Maybe something like "If you upgrade a user's type (via the UI or via the upgrade request email), their user type changes to that [level?] across all accounts in the organization." 

I understand that we have some specific rules about when/how we use 'upgrade', though.